src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -94,20 +94,24 @@ open class JicofoServices(
             }
         }
 
-        val focusJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString()
         conferenceRequestHandler = ConferenceRequestHandler(
             focusManager = focusManager,
-            focusAuthJid = focusJid,
+            focusAuthJid = XmppConfig.client.username.toString() + "@" + XmppConfig.client.domain.toString(),
```

Comment:
nit:  `"${XmppConfig.client.username}@${XmppConfig.client.domain}`

(I'm _pretty_ sure you don't need the `toString`s there when using that method, but not 100%)